### ISSUE TITLE
Heavy fps stutter whenever I mouseover on healing spec

### ISSUE THREAD

**Issue by Amortalist** [Hekili/hekili#602]
Using latest version available, this problem only happens on my Rdruid while the addon is loaded but not in use (healing spec), other specs / classes are ok. It is 100% tied to Hekili (tried every addons one by one). It's a similar type of stutter that when changing a talent on a dps spec. 

Basically whenever I mouseover a npc or a player, or a health bar in my groups, my fps drops heavily, independently of if I'm in combat or not. It is ONLY tied to healing specs, as dps specs doesn't seem to have this problem.

**Comment by Hekili** [comment]
Can you provide all the ticket information?  Obviously, a snapshot would not apply.

**Comment by Amortalist** [comment]
Hey, sorry not used to this : 

build: v9.0.2-1.0.13
level: 53 (53)
class: DRUID
spec: restoration

talents: none

pvptalents: none

covenant: night_fae

azerite: rampant_growth = 1
    resounding_protection = 1
    synergistic_growth = 1
    waking_dream = 1

essences: [none], none

sets/legendaries/artifacts: none

gear: corrosive_handlers_gloves = 1
    deadly_laestrite_choker = 1
    deathstalkers_shoulderpads = 1
    depraved_darkblades_belt = 1
    desolate_leather_armguards = 1
    desolate_leather_treads = 1
    glimmerhoof_drape = 1
    hood_of_the_hidden_path = 1
    infinitely_divisible_ooze = 1
    lakalis_spire_of_knowledge = 1
    loop_of_pulsing_veins = 1
    rainshadow_hauberk = 1
    seal_of_ghoulish_glee = 1
    skyterrors_stonehide_leggings = 1
    sunblood_amethyst = 1

legendaries: none

itemIDs: 143904, 159305, 159463, 166553, 172235, 172241, 173143, 177778, 178691, 178698, 178714, 178769, 178819, 178826, 182560

settings: aoe = 3
    buffPadding = 0
    custom1Name = Custom 1
    custom2Name = Custom 2
    cycle = false
    cycle_min = 6
    damage = true
    damageDots = false
    damageExpiration = 6
    damagePets = false
    damageRange = 0
    debuffPadding = 0
    enabled = true
    gcdSync = true
    maxRefresh = 10
    maxTime = 33
    nameplateRange = 8
    nameplates = true
    package = Guardian
    petbased = false
    potion = focused_resolve
    potionsReset = 20180919.1
    throttleRefresh = false
    throttleTime = false
    catweave_bear = false
    ironfur_damage_threshold = 5
    maul_rage = 20
    owlweave_bear = false

toggles: cooldowns = false
    custom1 = false
    custom2 = false
    defensives = true
    essences = true
    interrupts = true
    mode = automatic
    potions = false

No error message to snapshot.

**Comment by Amortalist** [comment]
It doesn't seem to be a conflict with another addon, the stutter still happens with only Hekili loaded.

**Comment by Hekili** [comment]
Are you comfortable testing an edit?

In Events.lua, we need to change line 2250; from:

```RegisterEvent( "UPDATE_BINDINGS", ReadKeybindings )
RegisterEvent( "PLAYER_ENTERING_WORLD", ReadKeybindings )
RegisterEvent( "ACTIONBAR_SLOT_CHANGED", ReadKeybindings )
RegisterEvent( "ACTIONBAR_SHOWGRID", ReadKeybindings )
RegisterEvent( "ACTIONBAR_HIDEGRID", ReadKeybindings )
RegisterEvent( "ACTIONBAR_PAGE_CHANGED", ReadKeybindings )
RegisterEvent( "ACTIONBAR_UPDATE_STATE", ReadKeybindings )
RegisterEvent( "SPELL_UPDATE_ICON", ReadKeybindings )
RegisterEvent( "SPELLS_CHANGED", ReadKeybindings )
```

To:

```RegisterEvent( "UPDATE_BINDINGS", ReadKeybindings )
RegisterEvent( "PLAYER_ENTERING_WORLD", ReadKeybindings )
-- RegisterEvent( "ACTIONBAR_SLOT_CHANGED", ReadKeybindings )
RegisterEvent( "ACTIONBAR_SHOWGRID", ReadKeybindings )
RegisterEvent( "ACTIONBAR_HIDEGRID", ReadKeybindings )
RegisterEvent( "ACTIONBAR_PAGE_CHANGED", ReadKeybindings )
RegisterEvent( "ACTIONBAR_UPDATE_STATE", ReadKeybindings )
RegisterEvent( "SPELL_UPDATE_ICON", ReadKeybindings )
RegisterEvent( "SPELLS_CHANGED", ReadKeybindings )
```

(It's just adding the `-- ` before the `ACTIONBAR_SLOT_CHANGED` line.)

Can you try that?  Save the file and then /reload in game.

**Comment by Amortalist** [comment]
Trying that asap

**Comment by Amortalist** [comment]
Yup, seems to be fixed ! Still having the stutter from changing talents but I guess it's normal because it has to calculate the new rotation.

**Comment by Amortalist** [comment]
Using this to ask a question, is there a way to show cooldowns separately from main bar ? For example to still have them displayed but not forcibly taking a spot in the rotation images ?

**Comment by Hekili** [comment]
No, not at the moment.

**Comment by Amortalist** [comment]
Is it planned ?

**Comment by Hekili** [comment]
Yes...ish.

I have a plan for a "tray" feature that basically does this, and can glow (or otherwise indicate) when the CD's criteria would be met.  (If the icons are showing all the time, it's not really a "priority" any more and this addon is a _priority_ helper.)

However, until that's ready, the smart/best option is to just get some WeakAuras for those key abilities that you want to be able to track all the time.

**Comment by Amortalist** [comment]
Awesome :) Yeah so far I'm mostly using the shortcut to activate it on boss / deactivate it on packs and world content. I'm already tracking most of them, I just enjoy the no brain mode with Hekili when playing alts. 

I also like the Bloodlust override feature, but sometimes it has to be delayed / it's off cd

**Comment by Hekili** [comment]
This is officially fixed in 1.0.14.  Thanks!
